contracts/fee_currencies.go

```diff
@@ -15,12 +15,19 @@ import (
 	"github.com/ethereum/go-ethereum/core/types"
 	"github.com/ethereum/go-ethereum/core/vm"
 	"github.com/ethereum/go-ethereum/log"
+	"github.com/ethereum/go-ethereum/params"
 )
 
 var feeCurrencyABI *abi.ABI
 
 var ErrFeeCurrencyEVMCall = errors.New("fee-currency contract error during internal EVM call")
 
+type CeloContractCaller interface {
+	bind.ContractCaller
+
+	Config() *params.ChainConfig
+}
+
```

Comment:
AFAICS, we could directly use `CeloBackend` instead of adding an additional interface. I'm not sure about the benefits of keeping the number of interfaces low vs separating interface from implementation in this case.

Reply:
Good point, I don't see a reason to add an interface for those two places, so let's use it directly.